src/screens/referScreen/referScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { Fragment } from 'react';
+import { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { FlatList, Text, View } from 'react-native';
+import { BasicHeader, Icon, MainButton, UserListItem } from '../../components';
+import get from 'lodash/get';
+// utils
+import { getReferralsList } from '../../providers/ecency/ecency';
+import { Referral } from '../../models';
+
+// styles
+import styles from './referScreenStyles';
+
+const ReferScreen = ({ navigation }) => {
+  const intl = useIntl();
+  const [referralsList, setReferralsList] = useState<Referral[]>([]);
+  useEffect(() => {
+    _getReferralsList();
+  }, []);
+
+  console.log('-----referralsList----- : ', referralsList);
+
+  const _getReferralsList = async () => {
+    const referralsListData = await getReferralsList('good-karma');
+    setReferralsList(referralsListData as any);
+  };
+  const _renderPointsEarned = () => {
+    return (
+      <View style={styles.pointsEarnedContainer}>
+        <Text style={styles.points}>1000</Text>
+        <Text style={styles.earendText}>Points Earned</Text>
+        <MainButton
+          // isLoading={isLoading}
+          // isDisable={isLoading}
+          style={styles.mainButton}
+          height={50}
+          onPress={() => console.log('pressed!')}
+        >
+          <View style={styles.mainButtonWrapper}>
+            <Text style={styles.unclaimedText}>Refer</Text>
```

Comment:
use intl and all caps REFER.. also instead of + icon how about we use a share icon